packages/sds/src/sds.ts

```diff
@@ -0,0 +1,162 @@
+import { sha256 } from "@noble/hashes/sha256";
+import { bytesToHex } from "@noble/hashes/utils";
+import { proto_sds_message } from "@waku/proto";
+
+import { DefaultBloomFilter } from "./bloom.js";
+
+export type Message = proto_sds_message.SdsMessage;
+export type ChannelId = string;
+
+export const DEFAULT_BLOOM_FILTER_OPTIONS = {
+  capacity: 10000,
+  errorRate: 0.001
+};
+
+const DEFAULT_CAUSAL_HISTORY_SIZE = 2;
+
+export class MessageChannel {
```

Comment:
is this class intended to be used by consumers? if so, can you generate `js doc` comments for `public` methods?

Reply:
It's likely there will be a `ParticipantState` class that holds one or more of these, but either way I will add some docs to the public methods